Sources/JavaScriptKit/JSObject.swift

```diff
@@ -57,4 +57,67 @@ public class JSObjectRef: Equatable {
     public func jsValue() -> JSValue {
         .object(self)
     }
+
+    public class func createTypedArray<Type>(_ array: [Type]) -> JSObjectRef where Type: TypedArrayElement {
+        let type: JavaScriptTypedArrayKind
+        switch Type.self {
+        case is Int8.Type:
```

Comment:
Please add protocol method as requirement of TypedArrayElement to convert type into JavaScriptTypedArrayKind instead of switch-case types. It would be more exhaustive